src/Components/NotificationIntegrationModal/TabComponent.jsx

```
@@ -0,0 +1,100 @@
+import { 
+  Typography, 
+  Box,
+  Button
+} from "@mui/material";
+import { useTranslation } from "react-i18next";
+import { useTheme } from "@emotion/react";
+import TextInput from "../../../src/Components/Inputs/TextInput";
+import Checkbox from "../../../src/Components/Inputs/Checkbox";
+
+const TabComponent = ({ 
+  type, 
```

ESLint

[error] 'type' is missing in props validation

(react/prop-types)